notebook/services/api/api.yaml

```diff
@@ -654,12 +639,18 @@ definitions:
       id:
         type: string
         format: uuid
-      notebook:
-        type: object
-        properties:
-          path:
-            type: string
-            description: path to notebook
+      path:
+        type: string
+        description: "The path to the session.  For console sessions, the path 
+         contains the cwd of the kernel and the name of the console session, 
+         which may be a uuid"
+      type:
+        type: string;
+        description: session type
+        enum:
+          - notebook
+          - file
+          - console
```

Comment:
I just noticed the `type` is declared as an enum in the spec. I think it's good that there's a declared list of types known by the notebook server, but I think it has to document that other types are allowed as well. I'm thinking of cases where people write JLab plug-ins that want to manage other types of sessions, and of apps like Eclair talking to KG, declaring their own session types.